src/app/static/jest.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     "globals": {
         "appUrl": "http://app",
         "ts-jest": {
-            tsConfig: 'tsconfig.json',
+            "tsconfig": 'tsconfig.json',
```

Comment:
opportunistic fix - jest was warning about `tsConfig` being deprecated in favour of `tsconfig`